src/cljstyle/config.clj

```diff
@@ -156,7 +156,13 @@
 ;; #### Rule: EOF Newline
 
 (s/def :cljstyle.config.rules/eof-newline
-  (s/keys :opt-un [:cljstyle.config.rules.global/enabled?]))
+  (s/keys :opt-un [:cljstyle.config.rules.global/enabled?
+                   :cljstyle.config.rules.eof-newline/trailing-blanks?]))
+
+
+;; whether to permit additional newline characters at the end of files
```

Comment:
I know I'm nitpicking, but please put this _above_ the `s/def` for the full `:eof-newline` config map to match the other sections.

Reply:
oh whoops thats right